rfcs/0093-rfc-categories.md

```diff
@@ -0,0 +1,152 @@
+---
+feature: rfc_categories
+start-date: 2021-05-19
+author: David Arnold
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @kevnicox, @Mic92, @gytis-ivaskevicius
+shepherd-leader: @kevincox
+related-issues:
+category: process
+---
+
+# Summary
+[summary]: #summary
+
+The RFC process is amended with means to categorize RFCs into one of: _feature_, 
+_information_ or _process_, where each category sets different accents that
+improve the overall outcome of the process.
+
+# Motivation
+[motivation]: #motivation
+
+## Some issues are not addressed by the community
+
+There is no appropriate venue for it. When reading the current RFC process, it becomes
+clear that it was made primarily with purely technical decisions in mind. It might only
+be a perception, but this perception systemically skews what's being discussed and what
+not.
+
+As a result, important topics might not be adressed and settled by the community at large
+in a generally accepted procedure.
+
+Specifically, it is hard to propose a well coordinated experiment across the community,
+document and acknowledge design issues, record proof-generated insight, but also to
+amend the RFC process itself, the forum rules, the code of conduct or propose any other 
+binding changes to community workflows or infrastructure.
```

Comment:
But actually, the RFC process has been changed without requiring categories, see e.g [RFC36](https://github.com/NixOS/rfcs/blob/master/rfcs/0036-rfc-process-team-amendment.md).

Reply:
I was briefly considering to replace the word "hard" with "unintuitive". But, then I figured they are just two nuances to convey the same meaning (=an obstacle).Maybe I should use "unintuitive"?

Reply:
I think this is the key. The fact that previous attempts have succeeded does not mean that the process is good. it just means that

1. There is sufficient desire.
2. The process is not literally impossible.

I agree that it is hard to find a quantitative number but the best we can do is look at examples where we feel that the process could have gone better for past RFCs whether or not the RFC was eventually accepted. And this approach still ignores RFCs that were never even submitted because the author couldn't fit their idea into the template.

I don't think "unintuitive" is the best word because in my mind the change here is intended to guide the review. Whereas "unintuitive" sounds like an author problem to me.